src/tests/ftest/datamover/dst_create.py

```diff
@@ -60,7 +60,7 @@ def run_dm_dst_create(self, tool, cont_type, api, check_props):
         pool1.connect(2)
 
         # Create a source cont
-        cont1 = self.create_cont(pool1, cont_type=cont_type)
+        cont1 = self.get_container(pool1, type=cont_type)
```

Comment:
(pylint-unexpected-keyword-arg) Unexpected keyword argument 'type' in method call